cmd/headscale/cli/generate.go

```diff
@@ -0,0 +1,41 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"tailscale.com/types/key"
+)
+
+func init() {
+	rootCmd.AddCommand(generateCmd)
+	generateCmd.AddCommand(generatePrivateKeyCmd)
+}
+
+var generateCmd = &cobra.Command{
+	Use:   "generate",
```

Comment:
[disclaimer from non-native speaker]

Would it be better just `create` ?

Reply:
I feel that create makes something in the database, while this generate is something that don't change the headscale instance or database